src/github.com/matrix-org/dendrite/clientapi/routing/register_test.go

```diff
@@ -0,0 +1,86 @@
+// Copyright 2017 Vector Creations Ltd
+// Copyright 2017 New Vector Ltd
```

Comment:
We don't ask you to assign copyright, and neither Vector Creations nor New Vector wrote this code, so the copyright belongs to you. You can either put your own name here or leave out the Copyright lines.

Reply:
(likewise you're free to add an assertion of your own copyright to any files you make non-trivial changes to. Not that I think it matters much from the legal point of view)